fat-jar/file-logback.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The short version:
logback.xml  is used as the bootstrap logging configuration. As soon as the config is parsed, if a logback file has been defined (and is writable), it reconfigures logback using file-logback.xml and the logback related config properties.

The long version:

Here's how it works with FatJar:
logback.xml is in the effective "-jar" classpath so the system bootstraps using this file from the root of the jar. The problem is that it's locked up in the jar and rather than requiring you to open the jar, edit and close back up again, it says, you configure the file name (and roll policy) in the opentsdb.conf and if it checks out, we'll reconfigure logback using file-logback.xml and the opentsdb config. Other supported options are:

1. Keep the console logging stream open after file logging has been enabled (it is closed by default)
2. Forget all that stuff and use this logback.xml over here.

The new config options for logback are:

* tsd.logback.file: The name of the log file the OpenTSDB logback configuration will log to when running the FatJar.
* tsd.logback.rollpattern: The logback rolling file appender roll pattern used to roll the file defined in tsd.logback.file.
* tsd.logback.console: If specified, logback will log to the configured file and keep logging to the console, otherwise, when the file appender is activated, the console appender is removed.
* tsd.logback.config: Points to an external [outside the FatJar] logback config file, in which case the internal file based logging configuration is ignored once the config has been loaded.

I'll submit an update to the docs for these options too. Need to be part of the pull request ?